mysql-test/suite/rocksdb/t/ttl_primary_read_filtering.test

```diff
@@ -0,0 +1,245 @@
+--source include/have_rocksdb.inc
+--source include/have_debug.inc
+
+--disable_warnings
+DROP TABLE IF EXISTS t1;
+--enable_warnings
+
+SET @@global.rocksdb_update_cf_options = 'default={disable_auto_compactions=true;};';
```

Comment:
Store the original value in a session variable and restore it at the end. If it changes, you don't need to update this code.